addons/io/org.openhab.io.hueemulation/src/main/java/org/openhab/io/hueemulation/internal/api/HueState.java

```diff
@@ -80,4 +84,13 @@ public String toString() {
         return "[on: " + on + " bri: " + bri + " hue: " + hue + " sat: " + sat + " xy: " + xyString + " ct: " + ct
                 + " alert: " + alert + " effect: " + effect + " colormode: " + colormode + " reachable: " + reachable;
     }
+
+    public boolean isOn() {
+      // The Light is on if Boolean variable received is true or we have a brightness
+      return this.on || this.bri > 0;
+    }
+
+    public int getBrightness() {
+      return this.bri < 1 ? 100 : (int) Math.ceil((double)this.bri / 255.0 * 100);
```

Comment:
Fix indentation and remove `this`:

```suggestion
        return bri < 1 ? 100 : (int) Math.ceil((double) bri / 255.0 * 100);
```